test/unit/cog_api/http/users_test.exs

```diff
@@ -50,4 +72,42 @@ defmodule CogApi.HTTP.UsersTest do
       end
     end
   end
+
+  describe "update" do
+    it "returns the updated user" do
+      cassette "users_update" do
+        params = %{
+          first_name: "Arnold",
+          last_name: "Vinick",
+          email_address: "arnold@example.com",
+          username: "arnie",
+          password: "12345",
+        }
+        endpoint = valid_endpoint
+        {:ok, user} = Client.user_create(endpoint, params)
+
+        params = %{first_name: "Arnie"}
```

Comment:
What do you think about keeping the tests similar in format between the fake and real?